Feature#908-Preklikavanie-roznych-grafov

jkmetko opened the pull request:
- uprava textu pre Y os v grafe
- pridanie previous buttonu
- uprava switch buttonu na next button

jkmetko committed:
Feature#909-Zjednotit-jazyk-aplikacie-do-AJ
- uprava textu pre Y os v grafe
- pridanie previous buttonu
- uprava switch buttonu na next button

AlexOstrovsky commented:
1. next tlacitko funguje spravne
2. previous tlacitko nefunguje a ostane mi ta ista hodnota
3. stale je to cez URL prerobit pretoze tak to nemoze ostat
4. vsimol som si, ze si upravil este v mergi pred tym schema.rb co je zle ale code reviewer si to asi nevsimol. Tabulky typu X:X tam nemaju vobec co robit.

jkmetko committed:
Feature#909-Zjednotit-jazyk-aplikacie-do-AJ
- fix pre previous button

jkmetko committed:
Feature#909-Zjednotit-jazyk-aplikacie-do-AJ
- zrusenie zasielania dat pomocou URL pre previous a next button
off the scope:
- zrusenie zasielania dat pomocou URL pre selecty
- spojazdnenie option selected v selectoch (ukazuju nazov aktualnej hodnoty na osi X a Y)

jkmetko committed:
Feature#909-Zjednotit-jazyk-aplikacie-do-AJ
- zrusenie zasielania dat pomocou URL pre previous a next button
- fix pre datasety bez ciselnych udajov
off the scope:
- zrusenie zasielania dat pomocou URL pre selecty
- spojazdnenie option selected v selectoch (ukazuju nazov aktualnej hodnoty na osi X a Y)

jkmetko commented:
2. opravene
3. opravene aj pre switchovanie columnov, aj pre menenie X a Y
4. ani neviem ako sa tam schema.rb dostala.... budem si na to davat pozor

jkmetko committed:
Feature#908-Preklikavanie-grafov

StankoPalko committed:
Update schema.rb
